quack_api/quack_api/Controllers/RecommenderController.cs

```diff
@@ -28,10 +28,14 @@ public RecommenderController(IOptions<RecommenderSettings> recommendersettingsAc
         protected IRecommenderService RecommenderService { get; set; }
 
         [HttpGet]
```

Comment:
Når der er en liste som parameter, så skal der bruges HttpPost i stedet for HttpGet